app/services/change_schedule.rb

```diff
@@ -0,0 +1,174 @@
+# frozen_string_literal: true
+
+class ChangeSchedule
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+  include ActiveModel::Validations::Callbacks
+  include Participants::ProfileAttributes
```

Comment:
could we avoid including profile attributes, as we will be deprecating all those when this work is complete?

Reply:
@cwrw All done tks 👍 😄